src/GitVersion.Core/Configuration/ConfigSerializer.cs

```diff
@@ -6,21 +6,21 @@ namespace GitVersion.Configuration;
 
 public class ConfigSerializer
```

Comment:
I would like everything related to serialization gone from Core. Not sure that's something we should do in this PR, but just FYI. Ideally, `GitVersion.Core` should not have any external dependencies.

```suggestion
public class ConfigurationSerializer
```

Reply:
> I would like everything related to serialization gone from Core. Not sure that's something we should do in this PR, but just FYI. Ideally, `GitVersion.Core` should not have any external dependencies.

Good point. I agree the core dll should not have so many dependencies. I think we should do this in a separate PR.